packages/vitest/src/node/reporters/default.ts

```diff
@@ -8,6 +8,11 @@ export class DefaultReporter extends BaseReporter {
   renderer?: ReturnType<typeof createListRenderer>
   rendererOptions: ListRendererOptions = {} as any
 
+  onPathsCollected(paths: string[] = []) {
+    if (paths.length === 1)
+      this.rendererOptions.renderSucceed = true
+  }
```

Comment:
We should probably undo the change if watch mode triggers rerun again but with >1 tests

Reply:
Done, Thank you for your review!